### ISSUE TITLE
Bootstrap theming support

### ISSUE THREAD

**Issue by lepipele** [btcpayserver/btcpayserver#118]
@NicolasDorier 

I've implemented foundation for theme support, once #117 is merged, I'll open new PR.

If you wanna try it out in the meantime and provide feedback, here is branch in my fork: https://github.com/lepipele/btcpayserver/tree/dev-bootstrap

**Comment by NicolasDorier** [comment]
nice nice I am so excited about this! :) I just merged.

**Comment by NicolasDorier** [comment]
Review, it seems good!

**Comment by lepipele** [comment]
Moving discussion to PR
